plugins/meta/tuning/tuning.go

```diff
@@ -239,11 +350,18 @@ func cmdAdd(args *skel.CmdArgs) error {
 	return types.PrintResult(tuningConf.PrevResult, tuningConf.CNIVersion)
 }
 
+// cmdDel will restore NIC attributes to the original ones when called
 func cmdDel(args *skel.CmdArgs) error {
-	// TODO: the settings are not reverted to the previous values. Reverting the
-	// settings is not useful when the whole container goes away but it could be
-	// useful in scenarios where plugins are added and removed at runtime.
-	return nil
+	tuningConf, err := parseConf(args.StdinData, args.Args)
+	if err != nil {
+		return err
+	}
+
+	err = ns.WithNetNSPath(args.Netns, func(_ ns.NetNS) error {
```

Comment:
now that I think about it, do we want to block deletion at all if this fails? When would there be an error that could be recoverable on retry?

Reply:
OK, I think you're right. Changed this to return always nil at the end, same as the 'stub' function that was here previously.

Reply:
Should this
```
tuningConf, err := parseConf(args.StdinData, args.Args)
	if err != nil {
		return err
	}
```

also be forced to return nil?